webapp/channels/src/components/channel_header_dropdown/menu_items/close_channel/close_channel.test.tsx

```diff
@@ -16,30 +16,36 @@ describe('components/ChannelHeaderDropdown/MenuItem.CloseChannel', () => {
         },
     };
 
-    it('should match snapshot', () => {
-        const wrapper = shallow<CloseChannel>(<CloseChannel {...baseProps}/>);
+    it("should match snapshot", () => {
+        const wrapper = shallow<typeof CloseChannel>(
```

Comment:
You can remove the type assertion.

```suggestion
        const wrapper = shallow(
```

Reply:
alright